### PR TITLE
Allow DB config during development

### PR DESCRIPTION
**Pull request by cmc333333**
Also updates the README.

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -2,13 +2,12 @@ default: &default
...
-  pool: 5
-
-development: *default
-test: *default
-production:
-  <<: *default
   username: <%= ENV['DB_USERNAME'] %>
   password: <%= ENV['DB_PASSWORD'] %>
   host: <%= ENV['DB_HOST'] %>
   port: <%= ENV['DB_PORT'] %>
```
Comment (afeld) [inline config/database.yml]: If the keys are set, do we have to pass explicit defaults? For example, do we need something like this?

```erb
<%= ENV['DB_USERNAME'] || root %>
```

Comment (phirefly) [inline config/database.yml]: I would prefer something like that

Comment (cmc333333) [inline config/database.yml]: An empty host and port definitely equate to the system defaults for postgres (localhost and.... 5321 -- something like that). I assume the username and password would be the same -- i.e. setting a blank `username` is equivalent to not setting one.

My setup is different than yours, would you mind confirming that a default is needed?

Comment (afeld) [inline config/database.yml]: Works On My Machine™

**Pull request merged**